client/post-editor/editor-drawer/index.jsx

```diff
@@ -240,7 +242,23 @@ const EditorDrawer = React.createClass( {
 					isNew={ this.props.isNew }
 				/>
 			</AccordionSection>
-		)
+		);
+	},
+
+	renderSeo: function() {
+		if ( ! config.isEnabled( 'manage/advanced-seo' ) || ! this.props.site ) {
```

Comment:
Hmm, after looking into it I think it needs to stay this way, because we need to check that the site has the business plan before we show the accordion. Another option would be to show this accordion for everyone, and show a promotional message if they don't have the business plan.